From 7e9a3513adeb8368faf737161027ceaecaf8e2c7 Mon Sep 17 00:00:00 2001
From: Huang Chenyi <b44402@freescale.com>
Date: Mon, 6 May 2013 12:53:32 +0800
Subject: [PATCH 193/602] ENGR00261406 uncomment "setprop
 vold.post_fs_data_done 1" in init.rc

If there is no fs-post-data action in the init.<device>.rc file,
you must uncomment this line,
otherwise encrypted filesystems won't work.

Signed-off-by: Huang Chenyi <b44402@freescale.com>
---
 imx6/etc/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index 392d01f..44d983a 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -256,7 +256,7 @@ on post-fs-data
     # must uncomment this line, otherwise encrypted filesystems
     # won't work.
     # Set indication (checked by vold) that we have finished this action
-    #setprop vold.post_fs_data_done 1
+    setprop vold.post_fs_data_done 1
 
 
 on boot
-- 
1.8.0

